### DISCUSSION TITLE
Automatic1111 in settings freezes

### DISCUSSION THREAD

**Discussion by sebaxakerhtc** [open-webui/open-webui#6670] [Issues]
# Bug Report

## Installation Method

docker

## Environment

- **Open WebUI Version:** v0.3.35

- **Operating System:** Windows 11
- **Browser** any

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [x] I have included the Docker container logs.
- [x] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

Going to settings to change the model, size, etc
Click save - all is good

## Actual Behavior:

Going to settings to change the model, size, etc
Click save - webui freezes and don't work anymore until I reload the page

## Description

**Bug Summary:**
This happend if at least once webui started without started automatic1111, and then never work until I rebuild a new webui container.
Interesting that image generation still works fine with the latest settings, but I want to change them!

## Reproduction Details

**Steps to Reproduce:**
Forget to start automatic1111
Go to settings => images
End
From now it will never work

## Logs and Screenshots

**Browser Console Logs:**

**Docker Container Logs:**
```
2024-11-03 11:49:10 INFO:     176.12.40.114:0 - "GET /api/v1/users/ HTTP/1.1" 200 OK
2024-11-03 11:49:12 INFO:     176.12.40.114:0 - "GET /api/webhook HTTP/1.1" 200 OK
2024-11-03 11:49:12 INFO:     176.12.40.114:0 - "GET /api/v1/auths/admin/config HTTP/1.1" 200 OK
2024-11-03 11:49:14 INFO:     176.12.40.114:0 - "GET /images/api/v1/config HTTP/1.1" 200 OK
2024-11-03 11:49:14 INFO:     176.12.40.114:0 - "GET /images/api/v1/image/config HTTP/1.1" 200 OK

2024-11-03 11:49:39 INFO  [open_webui.apps.images.main] Setting image model to SDXL-Turbo\dreamshaperXL_v21TurboDPMSDE.safetensors [4496b36d48]
```

**Screenshots/Screen Recordings (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Additional Information

[Include any additional details that may help in understanding and reproducing the issue. This could include specific configurations, error messages, or anything else relevant to the bug.]

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by GalenOfTheShadows** [comment]
I had this same trouble in Google Chrome(Linux), my solution was as follows:

1. Reduce my browser window size from 3840x2160 to 1920x1080 or less.
2. Ensure that the "Set CFG Scale variable was a float in the format of X.X, and not a pure integer (like X), even though every time I save the page, it resets to a pure integer.

It seems that the fullscreen on 4K was causing the error message about CFG Scale to display as a very thin line of the error dialog's background color at the very bottom of my browser tab (possibly because it was simply off the viewport, IDK), and thus, I could not see the rather cryptic message that the string was not an integer (note to dev: should probably write a clearer error message for this, explaining the expected format of the form field.)

Once I reduced my window size, the error dialog popped up towards the top of the page, and I was able to figure out what it was talking about, and thus found that the short display of the dialog was indicating the only float value on the page. I changed it from 7 => 7.0, and the settings successfully saved and I was good to go again. (I had to change our AUTOMATIC1111 Base URL due to some network changes happening. When I originally set up the integration, I was on my laptop with a 1920x1080 FHD panel. This time, I was on my workstation with 4K panels.)
